TypeCobol/Compiler/Nodes/Node.cs

```diff
@@ -717,6 +717,19 @@ public Node Get(string uri, int startIndex)
             return str.ToString();
         }
 
+        public override bool Equals(object obj)
+        {
+            return Object.ReferenceEquals(this, obj as Node);
+        }
+
+        public override int GetHashCode()
+        {
+            unchecked
+            {
+                return (13 * 397) ^ base.GetHashCode();
+            }
+        }
```

Comment:
Same remark as for StorageArea. No need to add code that does the same as the framework.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1600/commits/8b7686fa9f02c00e2a7150589c65d7fd66b787c8